docs/solana/speclanguage.rst

```diff
@@ -1,8 +1,138 @@
 Certora Verification Language for Rust (CVLR)
```

Comment:
If you rename the file from `.rst` to `.md` the spell checking warnings that are in code snippets won't fail. (this worked for me on https://github.com/Certora/Documentation/pull/337)